### PR TITLE
add createNestedViews example

### PR DESCRIPTION
**Pull request by bucko13**
An example for new utility: [createNestedViews](https://bpanel.org/docs/ui-utilities.html#createnestedviews)

### CODE REVIEW

**Comment by bucko13** [comment]
cc: @bantalek

File: lib/components/NestedViews.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <Text type="p">This is the "home view" that shows when at the parent path</Text>
+    <Text type ="p">Path: {location.pathname}</Text>
+  </div>
+);
+
+const NotFound = ({ location }) => (
+  <div>
+    <Header type="h3">404</Header>
+    <Text type="p">
+    You can optionally have a 404 view when you've navigated to an unregistered view.
```
Comment (tynes) [inline lib/components/NestedViews.js]: I wonder if escaping the `'` would help with the formatting on github

Comment (bucko13) [reply lib/components/NestedViews.js]: unfortunately it didn't work. Just output a `\` :-/ . I could put it in brackets and then use template literals I suppose.

File: lib/components/NestedViews.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <Text type="p">This is the "home view" that shows when at the parent path</Text>
+    <Text type ="p">Path: {location.pathname}</Text>
+  </div>
+);
+
+const NotFound = ({ location }) => (
+  <div>
+    <Header type="h3">404</Header>
+    <Text type="p">
+    {`You can optionally have a 404 view when you've navigated to an unregistered view.`}
```
Comment (bucko13) [inline lib/components/NestedViews.js:22]: Lol, well it works at least! @tynes

**Comment by bucko13** [comment]
@tynes made that formatting look better on github. Kind of annoying though :-/. Anything else for this?

**Review by tynes — APPROVED** [review]

**Pull request merged**